packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -219,7 +226,7 @@ class Route extends EmberObject implements IRoute {
     }
 
     let transition = this._router._routerMicrolib.activeTransition;
-    let state = transition ? transition.state : this._router._routerMicrolib.state;
+    let state = transition ? transition[STATE_SYMBOL] : this._router._routerMicrolib.state;
```

Comment:
Need svelting here I think

Reply:
I think you might be confused. `transition.state` is still needed internally but access to it from application code should be deprecated and we just tell people to use the `RouteInfo`. All of these symbols are just allowing the integration of ember-router and router.js to work. I could have simply renamed the method but I felt that using a faux-symbol may deter people further. In the future we should just make these real symbols.

Reply:
👍 	thank you!

_some_ of the comments though are specifically about deprecations, and should still apply